### ISSUE TITLE
arm64: dhcpcd issue

### ISSUE THREAD

**Issue by arm64b** [linuxkit/linuxkit#2305]
**Description**
On ARM64 platform, with the mainline code to build all the required components by linuxkit.yml except the nginx, then 
`$bin/moby build linuxkit.yml
$bin/linuxkit run linukit`

Can't boot to the shell, hang on:
Welcome to LinuxKit

                        ##         .
                  ## ## ##        ==
               ## ## ## ## ##    ===
           /"""""""""""""""""\___/ ===
          {                       /  ===-
           \______ O           __/
             \    \         __/
              \____\_______/

2017/07/27 10:38:53 Error running 002-dhcpcd: fork/exec /usr/bin/runc: too many levels of symbolic links
/bin/rc.init: line 124: /etc/init.d/020-containerd: Symbolic link loop
 

**Steps to reproduce the issue:**

**Describe the results you received:**

**Describe the results you expected:**

**Additional information you deem important (e.g. issue happens only occasionally):**

**Comment by arm64b** [comment]
Fortunately, I have backed up a working branch. Will triage tomorrow when I am back to the office

**Comment by justincormack** [comment]
Are you using the latest version of the `moby` tool? There were some changes there that may well affect this.

latest is currently
```
moby version
moby version 0.0
commit: 36217e5145f1e54807490c09cc389f5e3ac99075
```

**Comment by ijc** [comment]
FWIW I just ran `linuxkit.yml` on #2306 (master + containerd bump which wouldn't impact this stuff) and it is fine. Here are my relevant paths/links:
```
(ns: getty) linuxkit-1afe5147afe9:~# ls -l /proc/1/root/etc/init.d/* /proc/1/root/usr/bin/onboot /proc/1/root/usr/bin/service /proc/1/root/bin/rc.*
-rwxr-xr-x    1 root     root          4186 Jul 26 15:02 /proc/1/root/bin/rc.init
-rwxr-xr-x    1 root     root           400 Jul 26 14:53 /proc/1/root/bin/rc.shutdown
-rwxr-xr-x    1 root     root           200 Jun 12 13:41 /proc/1/root/etc/init.d/000-issue
lrwxrwxrwx    1 root     root            15 Jul 26 15:47 /proc/1/root/etc/init.d/010-onboot -> /usr/bin/onboot
lrwxrwxrwx    1 root     root            16 Jul 27 10:00 /proc/1/root/etc/init.d/020-containerd -> /usr/bin/service
-rwxr-xr-x    1 root     root       1843496 Jul 26 15:47 /proc/1/root/usr/bin/onboot
-rwxr-xr-x    1 root     root      13163976 Jul 27 10:00 /proc/1/root/usr/bin/service
```

**Comment by arm64b** [comment]
@justincormack  I am working on the mainline arm64 support, so all the code is cloned from the master branch for linuxkit git tree, so the bin/moby and bin/linux also does, below is the output of the moby version:
`# bin/moby version
moby version 0.0
commit: 36217e5145f1e54807490c09cc389f5e3ac99075
`
I heard that @rn has already got an arm64 platform, so maybe he can reproduce this if possible, I will continue to check the gap between the mainline and my working branch in terms of arm64 support.

**Comment by arm64b** [comment]
closed with #2332
